compiler/crates/common/src/feature_flags.rs

```diff
@@ -124,6 +124,10 @@ pub struct FeatureFlags {
     /// Disable validation of the `edgeTypeName` argument on `@prependNode` and `@appendNode`.
     #[serde(default)]
     pub disable_edge_type_name_validation_on_declerative_connection_directives: FeatureFlag,
+
+    // Removes the @ts-nocheck from generated Typescript Files
```

Comment:
Use `///` (three slashes) here to have Rust pick this up as a documentation comment. Should cause it to get included in the compiler-config-schema.

Also can we invert this naming? I think we want to end with checking ts files being the default, and defaulting flags to `false` is generally more intuitive. So, maybe we add this flag as `typescript_disable_checking_generated_files`?

Reply:
Then should we update all test snapshots to remove the nocheck? 

As it stands today, we'd break many TS implementations if we release with defaulting to checking TS.

Another question - do we prefer this type to be `bool` or `FeatureFlag`?

Reply:
If it would break many TS implementations today, then maybe we could _start_ with it defaulting to true and then switch it to defaulting to false once we think we are in a place where it would work for most people?

I think `bool` is probably okay, but `FeatureFlag` would let you do a setup where you opt-out on a per-[something] basis by specifying a list of strings in the config. Could be fragment/query/mutation names, or some other pattern. Can be useful if you think people will need to do an incremental adoption.

Reply:
> If it would break many TS implementations today, then maybe we could start with it defaulting to true and then switch it to defaulting to false once we think we are in a place where it would work for most people?

This sounds good to me! I haven't found any examples in the existing where we default to true, how would I add that?